client/src/main/java/io/pravega/client/stream/impl/ReaderGroupStateManager.java

```diff
@@ -114,7 +113,7 @@ void initializeReader() {
             throw new IllegalStateException("The requested reader: " + readerId
                     + " cannot be added to the group because it is already in the group. Perhaps close() was not called?");
         }
-        acquireTimer.zero();
+        acquireTimer.reset(Duration.ofMillis(initialAllocationDelay));
```

Comment:
I added a random element. We don't want to let it get close to zero, because the problem we are trying to solve to give other readers time to startup before claiming segments.